google/ggrc-core
# GGRC-6067 Add test for checking is_latest_revision

# Issue description

Snapshot has flag "is_latest_revision" == False but, it matchs latest revision

# Steps to test the changes

1 Log in.
2 Create a snapshot.
3 Update original object.
4 Update snapshot to the latest version.
5 See that snapshot has is_latest_revision flag == True.

# Solution description

Added new integration test that checks the behavior of is_last_revision flag

# Sanity checklist


# PR Review checklist
